tests/test_text/test_dispersion.py

```diff
@@ -67,12 +67,27 @@ def test_dispersionplot_generator_input(self):
         """
         corpus = self.load_data('hobbies')
 
-        text = chain(*map(lambda s: s.split(), corpus.data))
+        text = (doc.split() for doc in corpus.data)
```

Comment:
We do still need the `map` here to ensure a generator is passed in; this line of code is creating a tuple in memory. Would you please change it to:

```python
text = map(lambda doc: doc.split() for doc in corpus.data)
```

Reply:
Yes, you're right - sorry; I mistook the parens for a tuple comprehension, but that does indeed wrap a generator expression. Might be a good comment, but you're right, go ahead and merge this in!

https://stackoverflow.com/questions/16940293/why-is-there-no-tuple-comprehension-in-python